manageiq-appliance_console.gemspec

```diff
@@ -25,11 +25,13 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "awesome_spawn",           "~> 1.6"
   spec.add_runtime_dependency "bcrypt",                  "~> 3.1.10"
   spec.add_runtime_dependency "bcrypt_pbkdf",            ">= 1.0", "< 2.0"
+  spec.add_runtime_dependency "ed25519",                 ">= 1.2", "< 1.3"
```

Comment:
Out of curiosity why is this explicitly needed?

Reply:
```
E, [2024-04-16T13:41:31.316966 #670326] ERROR -- net.ssh.authentication.agent[b18]: ignoring unimplemented key:unsupported key type `ssh-ed25519'
net-ssh requires the following gems for ed25519 support:
 * ed25519 (>= 1.2, < 2.0)
 * bcrypt_pbkdf (>= 1.0, < 2.0)
See https://github.com/net-ssh/net-ssh/issues/565 for more information
Gem::LoadError : "ed25519 is not part of the bundle. Add it to your Gemfile."
```

Reply:
ohhh interesting - wonder if we can drop the rbnacl gem then?

Reply:
See https://github.com/ManageIQ/manageiq-appliance_console/commit/67653544afebe7bdbfedeed9c9161cac38592aeb - I think they were only added for this specific cipher.

Getting rid of those would be huge cause they require a C lib.

Reply:
Added in 67653544afebe7bdbfedeed9c9161cac38592aeb

Looks like yes, that was the only reason rbnacl was added

We might be able to drop rbnacl-devel rpm also: 350c879fe1f080bf154bea3e0616d51b0c0d9ee6

Reply:
I think we're okay actually, `jwt` just needs libsodium not `rbnacl`: https://github.com/ManageIQ/manageiq-rpm_build/pull/117#issuecomment-724800717

Reply:
yeah it probably should have been there all along in both places.

Reply:
Agreed, https://github.com/ManageIQ/manageiq-providers-google/pull/264

Reply:
Removed here https://github.com/ManageIQ/manageiq-appliance_console/pull/248/files#diff-5cfba543196ff7509ccbd7b987182595a8baa8cc3e4a4e1040e20ccfd75d885eL36